Makefile

```diff
@@ -8,4 +8,14 @@ install:
 		libboost-thread-dev \
 		libtbb-dev \
 		python3-dev \
-		gdal
\ No newline at end of file
+		libgdal-dev \
+		gdal-bin
```

Comment:
These two libraries (libgdal-dev, gdal-bin) seem to be needed to install this inside a container, but I'm not convinced that this change needs to happen here in the Makefile.

The Dockerfile has its own `apt install` steps, and the packages have been copy-pasted from here so as to take advantage of Docker's layer caching. If the Dockerfile installs via `RUN make install` it will execute each time the image builds, whereas if we use `RUN apt install ...` the layer will cache, resulting in faster builds.

Since the Dockerfile has its own set of requirements, the Makefile could be considered a tool for localhost Linux installs, and this change could be reverted.